### DISCUSSION TITLE
blog/ai-ppt-to-hugo/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#359] [Announcements]
# blog/ai-ppt-to-hugo/

使用通义千问 AI 快速生成 PPT，并通过 Canva 调整后嵌入 Hugo 网站，展示高效方法。

https://jimmysong.io/blog/ai-ppt-to-hugo/

**Comment by rootsongjc** [comment]
通义千问 AI PPT 的效果还是不错的，如果能再加上个网页嵌入功能就更好了，或者与 Canva 合作也行。

**Reply by rootsongjc** [reply]
不过我现在更推荐直接使用 Marp 来生成 PPT 和托管在本站上了
